Yosemite/Yosemite/Model/Extensions/URL+Helpers.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  URL+Helpers.swift
+//  Yosemite
+//
+/// Convenient extension methods for URL
+import Foundation
+
+extension URL {
+    static public func getWooCommerceAdminURL(from storesManager: StoresManager) -> URL {
```

Comment:
In this case I think the prefix `getWooCommerce` is a little bit redundant. As we discussed, the prefix get is not very used in Swift or specifically in our codebase for methods, you can directly call the method after the attribute. Regarding WooCommerce, I think it is not necessary at this point to add it, as we know that the admin URL is related to WooCommerce. So summing up, I would have added something like:

`func adminURL(from storesManager: StoresManager) -> URL
`
or, to add more context about what we are actually doing there:

`func safeAdminURL(from storesManager: StoresManager) -> URL
`
or more explicitly:

`func adminURLWithFallback(from storesManager: StoresManager) -> URL`